migrations/lib/importer/mapping_type.rb

```diff
@@ -6,6 +6,7 @@ module MappingType
     BADGES = 2
     CATEGORIES = 3
     GROUPS = 4
+    BADGE_GROUPINGS = 5
```

Comment:
You already used 5 for tags

Reply:
Thanks! I’ve got a bunch of these going on at the same time, in no particular order so I didn't want to leave gaps. I should have fixed this before marking it ready for the ready for review.